public/javascripts/data/paragraph.js

```diff
@@ -1,30 +1,27 @@
 const Database = require('better-sqlite3');
 
-
 // Function writes paragraph to the database
-exports.insertParagraph = (paragraph) => {
+exports.insertParagraph = (paragraph, userId) => {
 
-    const db = new Database('public/javascripts/data/stories.db');
-    // TODO:
-    // 1. (user_id, stories_id) should contain current user and current story
-    
-    const new_paragraph = db.prepare('INSERT INTO paragraphs (user_id, stories_id, content) VALUES (?, ?, ?)');
-    const values = new_paragraph.run(1, 1, paragraph);
+    let db = new Database('public/javascripts/data/stories.db');
```

Comment:
Controller:
```js
// controllers/stories.js
// POST /add-story
exports.addStory = (req, res) => {
  paragraph_content = req.body.content
  userId = req.session.user.id
  storiesManager.addStory(content, userId)
}
```

storiesManager.js
```js
exports.addStory = (content, userId) => {
  story = storyData.add_story(userId)
  paragraphData.insertParagraph(content, userId, story.id)
}
```

Reply:
I ended up taking these changes (add and display stories )to a new branch. Since I'm going to be changing the routes, I wanted to start on a fresh branch.

Reply:
A good call. Keep your branches related to a specific set of changes. Otherwise, your pull requests can get too large and difficult to review